target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -283,6 +287,12 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan" "3:lan"
 		;;
+	qxwlan,e558-v2-8m|\
+	qxwlan,e558-v2-16m)
+		ucidef_set_interfaces_lan_wan "eth1" "eth0"
```

Comment:
If the proper tagged variant of ucidef_add_switch is used, this line here can be dropped entirely.